net/atftp/files/atftpd.init

```diff
@@ -6,6 +6,10 @@ PIDFILE=/tmp/run/atftpd.pid
 
 start() {
 	config_load atftpd
+	config_get ENABLE service enable 0
```

Comment:
Make variable `ENABLE` to lowercase `enable`. This is more common in this repository

Reply:
Should I then refactor whole package? Since SRV and PORT and using uppercase also.

Reply:
I didn't see that at all. I just looked at your change. If that's the case, then please yes.
Do this in a separate commit before your changes.

Reply:
Done.
I have not tested this new series but there shouldn't be possible in such simple script.
Implemented refactor + base change of this pull request.